docs/software/communication/openmpi.md

```diff
@@ -6,5 +6,53 @@ However, [OpenMPI](https://www.open-mpi.org/) can be used as an alternative in s
 
 To use OpenMPI on Alps, it must be built against [libfabric][ref-communication-libfabric] with support for the [Slingshot 11 network][ref-alps-hsn].
 
+## Using OpenMPI
+
+!!! warning
+    Building and using OpenMPI on Alps is still [work in progress](https://eth-cscs.github.io/cray-network-stack/).
+    The instructions found on this page may be inaccurate, but are a good starting point to using OpenMPI on Alps.
+
+!!! todo
+    Deploy experimental uenv.
+
 !!! todo
-    Building OpenMPI for Alps is still work in progress: https://eth-cscs.github.io/cray-network-stack/.
+    Document OpenMPI uenv next to prgenv-gnu, prgenv-nvfortran, and linalg?
+
+OpenMPI is provided through a [uenv][ref-uenv] similar to [`prgenv-gnu`][ref-uenv-prgenv-gnu].
+Once the uenv is loaded, compiling and linking with OpenMPI and libfabric is transparent.
+At runtime, some additional options must be set to correctly use the Slingshot network.
+
+First, when launching applications through slurm, [PMIx](https://pmix.github.com) must be used for application launching.
+This is done with the `--mpi` flag of `srun`:
+```bash
+srun --mpi=pmix ...
+```
+
+Additionally, the following environment variables should be set:
+```bash
+export PMIX_MCA_psec="native" # (1)
+export FI_PROVIDER="cxi" # (2)
+export OMPI_MCA_pml="^ucx" # (3)
+export OMPI_MCA_mtl="ofi" # (4)
+```
```

Comment:
```
export PMIX_MCA_psec="native" # (1)!
export FI_PROVIDER="cxi" # (2)!
export OMPI_MCA_pml="^ucx" # (3)!
export OMPI_MCA_mtl="ofi" # (4)!
```
Use `!` to hide the `#` symbol. Maybe worth a comment bout `^` in the node (to explain it is a negation)?

Reply:
```suggestion
```bash
export PMIX_MCA_psec="native" # (1)!
export FI_PROVIDER="cxi" # (2)!
export OMPI_MCA_pml="^ucx" # (3)!
export OMPI_MCA_mtl="ofi" # (4)!
```
```

Reply:
> Maybe worth a comment bout ^ in the node (to explain it is a negation)?

Good idea. Added some text in https://github.com/eth-cscs/cscs-docs/pull/75#discussion_r2028583901.